gradio/blocks.py

```diff
@@ -65,14 +67,17 @@ def set_event_trigger(
         if not isinstance(outputs, list):
             outputs = [outputs]
 
-        Context.root_block.fns.append((fn, preprocess, postprocess))
+        Context.root_block.fns.append(BlockFunction(fn, preprocess, postprocess))
         Context.root_block.dependencies.append(
             {
                 "targets": [self._id] if not no_target else [],
                 "trigger": event_name,
                 "inputs": [block._id for block in inputs],
                 "outputs": [block._id for block in outputs],
                 "queue": queue,
```

Comment:
how queue is used within frontend? is it currently used?

Reply:
yes queueing should work as expected